src/Microsoft.DocAsCode.DataContracts.Common/TocItemViewModel.cs

```diff
@@ -23,6 +23,10 @@ public class TocItemViewModel
         [JsonProperty("name")]
         public string Name { get; set; }
 
+        [ExtensibleMember]
+        [JsonExtensionData]
+        public Dictionary<string, object> Metadata { get; set; }
+
```

Comment:
we have *additional* for yaml extension property, and another for json extension property.